### ISSUE TITLE
Feat: Create COMMIT_MESSAGE_STYLE_GUIDE.md for projects to refer to.

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/.github#3]
### Is your feature request related to a problem? Please describe.
Currently projects under anita-org has their own copy of `Commit Message Style Guide` instructions on wiki page. These pages are redundant because we only need one page that all projects can refer to.
 
### Describe the solution you'd like
Create COMMIT_MESSAGE_STYLE_GUIDE.md template.

### Describe alternatives you've considered
NA

### Additional context
NA

**Comment by mtreacy002** [comment]
@isabelcosta, please confirm that this issue is valid and perhaps we can open it for OSH contributors.

**Comment by epicadk** [comment]
I think we decided against this in the QA open session because it can be pretty challenging to rebase commits for first timers. Instead it is upto the maintainer to give an appropriate name to the commit while merging.

**Comment by isabelcosta** [comment]
We can still do this, but i would move this to documentation repository instead. This is because this repository is meant for community health profile documents. Such as code of conduct, that can be picked up by GitHub
As per the documentation https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file:

> GitHub will use and display default files for any public repository owned by the account that does not have its own file of that type in any of the following places: (...)

@mtreacy002 do you mind if we move the files you suggested, that is not picked up by GitHub, to `documentation` repository instead? we can transfer issues

**Comment by epicadk** [comment]
> We can still do this, but i would move this to documentation repository instead. This is because this repository is meant for community health profile documents. Such as code of conduct, that can be picked up by GitHub
> As per the documentation https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file:
> 
> > GitHub will use and display default files for any public repository owned by the account that does not have its own file of that type in any of the following places: (...)
> 
> @mtreacy002 do you mind if we move the files you suggested, that is not picked up by GitHub, to `documentation` repository instead? we can transfer issues

Agreed a lot of docs not related to the project have been merged (talking about mentorship android in particular). We don't really need them in the repo and instead they can just be on the documentation repo.

**Comment by mtreacy002** [comment]
Sure, @isabelcosta , I wasn't sure the difference between documentation and .github repos before, so thanks for making it clear for me and sorry for the error 😊. Feel free to move other issues I just opened to documentation repo 😉

**Comment by isabelcosta** [comment]
No problem at all @mtreacy002 I am also getting acquainted with `.github` repository, on how it works :)

**Comment by mtreacy002** [comment]
Btw, @isabelcosta,w will you be able to transfer the issues between repos with a click or should I closed the issues from this repo and open them in the documentation repo manually?

**Comment by isabelcosta** [comment]
ah, I can do it @mtreacy002 I thought you could do it
